src/utils/transaction.js

```diff
@@ -12,4 +12,35 @@ export const generateTransaction = account => ({
   }],
 })
 
+export const generateRentTransaction = (accountName, propertyName, serverKey, userKey, serverAuth, userAuth) => {
```

Comment:
Now that there are more than one transactions the above should be specified as `generateLikeTransaction` for clarity